store/activities.js

```diff
@@ -14,6 +14,11 @@ const config = [
     path: 'deposit-dates',
     icon: 'calendar-check',
   },
+  {
+    id: 'dois',
+    path: 'dois', // TODO: Find good icon
```

Comment:
I would use either [barcode](https://iconify.design/icon-sets/mdi/barcode.html) or [QR-code](https://iconify.design/icon-sets/mdi/qrcode.html). Also, we may use [widgets](https://iconify.design/icon-sets/mdi/widgets.html) that means a registry in Windows but this is not as clear as previous ones.

Reply:
Perhaps I would go for barcode but it still doesn't explain the content of the tab to me.

Reply:
Also, I thought about drawing something similar or getting actual favicon from doi.org. However, for this we might have to write them about asking a permission.

DOI.org does not have any branding resources and policy of using them on the website unfortunately and this process may take a long time.

@mcancellieri do you have any recommendation in the process? Should we write to DOI.org?

Reply:
i like the barcode one.
To be honest, I have never seen the favicon of doi.org used anywhere else so it will be as informative as the barcode, so I vote for the barcode.